### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by apalizha**
fix the bulk_db's by removing duplicated bulk entries. Also updated corresponding the mapping of index to mpid and precomputed surface structures files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,25 +69,31 @@ We generated around 2.5M adsorbate+catalyst inputs which were then deduplicated
...
+(uncompressed size 18GB). It has 11k pickle files, with filename format `<zero-based-index>.pkl`, one for each of the bulk materials based on the bulk indices.
+
+
+**Note**
+
+OC20 was generated with an older version of the bulks (https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/bulk_db_nelems_2020may12.pkl (MD5 checksum: `b87281d01a2006586fa719a1d2ee2682`), which has 50 duplicated bulk entry that is now removed in the latest version. 
 
-Note that uncompressing this file will result in the folder `precomputed_structure_info/`
-(uncompressed size 18GB). It has 11500 pickle files, with filename format `<zero-based-index>.pkl`, one for each of the 11500 bulk materials.
+and an older version of the adsorbates (https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/adsorbate_db_2020may12.pkl, MD5 checksum: `6c553a7c2009e11612c247ffb813fd74`)), which contains an inconsistency that is fixed in the latest version.
```
Comment (janiceblue) [inline README.md:96]: nit: you have two closing )), and grammar wise either combine this with previous sentence or start a new sentence

File: README.md [hunk 2]
```diff
@@ -69,25 +69,31 @@ We generated around 2.5M adsorbate+catalyst inputs which were then deduplicated
...
-To speed up surface sampling from a chosen bulk material we precomputed surface enumerations. These can be found here: https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/precomputed_structure_info_oc20.tar.gz (5.6GB, MD5 checksum: `7b2d4ffab4f45beb39f1b07493425386`)
+To speed up surface sampling from a chosen bulk material we precomputed surface enumerations. These can be found here: https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/precomputed_surfaces_2021june11.tar.gz (5.5GB, MD5 checksum: `795721f325b09202729c8f423a3c4d5c`)
+
+Note that uncompressing this file will result in the folder `precomputed_surfaces_2021june11/`
+(uncompressed size 18GB). It has 11k pickle files, with filename format `<zero-based-index>.pkl`, one for each of the bulk materials based on the bulk indices.
+
+
+**Note**
+
+OC20 was generated with an older version of the bulks (https://dl.fbaipublicfiles.com/opencatalystproject/data/input_generation/bulk_db_nelems_2020may12.pkl (MD5 checksum: `b87281d01a2006586fa719a1d2ee2682`), which has 50 duplicated bulk entry that is now removed in the latest version.
```
Comment (janiceblue) [inline README.md:94]: nit: "50 duplicated bulk entries that are"

**Review by janiceblue — APPROVED** [review]
LGTM

**Comment by apalizha** [comment]
fixed the commented grammar/formatting error.

**Pull request merged**